### PR TITLE
Add ability to watch / autoreload in development mode

### PR DESCRIPTION
**Pull request by saghul**
Fixes: https://github.com/jitsi/jitsi-meet-electron/issues/103

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -25,6 +25,14 @@ autoUpdater.logger.transports.file.level = 'info';
  */
 require('electron-debug')({ showDevTools: false });
 
+/**
+ * Enable automatic reloads in development mode.
+ */
+
```
Comment (akshitkrnagpal) [inline main.js]: style: spurious empty line :stuck_out_tongue_closed_eyes:

File: main.js [hunk 2]
```diff
@@ -25,6 +25,14 @@ autoUpdater.logger.transports.file.level = 'info';
...
+if (isDev) {
+    require('electron-reload')(path.join(__dirname, 'build'));
```
Comment (akshitkrnagpal) [inline main.js]: How about moving `require('electron-debug')` to this block too?

**Comment by akshitkrnagpal** [comment]
Hey @saghul 
It doesn't automatically open window?

File: main.js (hunk 1 above)
Comment (saghul) [reply main.js]: I taught you well :-)

**Comment by akshitkrnagpal** [comment]
>  You mean there is a way to do this in a single command?

Yeah. I got it working. But that would only work for react stuff.

**Comment by akshitkrnagpal** [comment]
LGTM.. !! 😄

**Pull request merged**